src/h3lib/include/bbox.h

```diff
@@ -37,6 +37,8 @@ bool bboxIsTransmeridian(const BBox* bbox);
 void bboxCenter(const BBox* bbox, GeoCoord* center);
 bool bboxContains(const BBox* bbox, const GeoCoord* point);
 bool bboxEquals(const BBox* b1, const BBox* b2);
-int bboxHexRadius(const BBox* bbox, int res);
+int bboxHexEstimate(const BBox* bbox, int res);
```

Comment:
Hextimate? ;)

Reply:
Aren't you aware that this is SRS BZNS? ;)